### PR TITLE
Releng sync

### PR DESCRIPTION
**Pull request by ermirizio**
Update/Sync Releng templates for tyk

:robot: Beep, boop, this PR has been generated by [automation](https://github.com/TykTechnologies/gromit/blob/master/cmd/policy.go) on Mon Oct 10 16:46:49 UTC 2022

This PR updates the release engineering specific code in the tyk repo,
and has been generated because some releng code was changed, and
the corresponding templates were updated.

These changes will mostly be limited to the `ci/` directory and `.github` directory.

If you think this was made by mistake, or if merging this might cause any instabilities,
please log a jira ticket in the TD project citing this PR.

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4338/merge
Commit:  
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3220995479)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4338/merge
Commit: 6990546294e0bc98805b7c7c730a4bd4c98b4261 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3221087543)

File: ci/goreleaser/goreleaser-el7.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # Generated by: gromit policy
-# Generated on: Thu Aug 25 13:46:43 UTC 2022
+# Generated on: Mon Oct 10 17:26:08 UTC 2022
 
 # Check the documentation at http://goreleaser.com
 # This project needs CGO_ENABLED=1 and the cross-compiler toolchains for
 # - arm64
 # - amd64
 
+
```
Comment (titpetric) [inline ci/goreleaser/goreleaser-el7.yml:9]: Unnecessary whitespace

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4338/merge
Commit: 30951e8ac17d147c311fc82dc83871fca37a6c34 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3221111442)

**Review by titpetric — APPROVED** [review]
There's some unnecessary whitespace which we could address, otherwise it seems like an expected version bump. 👍

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4338/merge
Commit: 4026c828dba6f56a9fb3d7fca9255f6175712184 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3221131420)

File: ci/goreleaser/goreleaser-el7.yml (hunk 1 above)
Comment (ermirizio) [reply ci/goreleaser/goreleaser-el7.yml:9]: will fix :) thx

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4338/merge
Commit: 0ccc4af97706f8bdfb92a9822e1bc2ce2d7f72f5 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3221204256)

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/4338/merge
Commit: 64771446d96d88cee625b827f6d1e7aa6edb1eae 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3221250644)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4338/merge
Commit: f28d526d89ef393248cbd900af1f1ff3e016611e 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3221322495)

**Review by buger — APPROVED** [review]

**Review by asutosh — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4338/merge
Commit: cceaff16a069cb2e38015f72e4b3e02df1c97b6d 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3221342574)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: a11aa851ea7859578c4e4143ef48820a9a4a09dd Releng sync (#4338)

* Releng sync

Co-authored-by: Gromit <policy@gromit>
Triggered by: push (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3221708668)
